src/main/java/io/kontur/disasterninja/controller/BivariateMatrixController.java

```diff
@@ -26,7 +26,7 @@ public class BivariateMatrixController {
 
     @Operation(summary = "Calculate data for bivariate matrix using insights-api service",
             tags = {"Bivariate matrix"},
-            description = "Calculate bivariate matrix for given geometry and important layers.")
+            description = "Calculate bivariate matrix for given geometry and important layers (hardcoded on front end). Performs GraphQL query PolygonStatistics/bivariateStatistic. 'name' in the response corresponds to 'param_id'")
```

Comment:
```suggestion
            description = "Calculate bivariate matrix for given geometry and important layers (hardcoded on front end, UI in US1414). Performs GraphQL query PolygonStatistics/bivariateStatistic. 'name' in the response corresponds to 'param_id'")
```

Reply:
User Story 1414 https://kontur.fibery.io/Tasks/User_Story/Add-analytical-parameter-to-favorites-1414